src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -229,6 +229,9 @@ class PythonSetup(Subsystem):
             `{'data-science': '3rdparty/data-science-constraints.txt'}`.
             If a resolve is not set in the dictionary, it will not use a constraints file.
 
+            You can use the key `__all__` to use the same constraints file for all resolves.
```

Comment:
It appears `__all__` means `__default__` which is not `__defaults__` (since its plural), but ... perhaps maddening for a 1st time user / reader? It's already a bit insane for that person that some config is toml and some BUILD - why not pick one!? But this might add insult to injury. A very similar concept is being spelled differently - seemingly gratuitously - in two different spots where you configure things.

Reply:
Excellent point, thank you for the suggestion.

The name `__default__` might also shed some insight on the question "What if I don't use resolves": https://docs.google.com/document/d/1HAvpSNvNAHreFfvTAXavZGka-A3WWvPuH0sMjGUCo48/edit#heading=h.n451mfzergzm. I'm going to wait to merge this until I figure that out.